bazel/v8.patch

```diff
@@ -101,6 +101,14 @@ index 0539ea176ac..92c7aeb904f 100644
                  "-isystem .",
              ],
              "//conditions:default": [],
+@@ -141,6 +144,7 @@ def _default_args():
+                 "-Wno-return-type",
```

Comment:
as its not tested might be best to comment that these args are required for arm - otherwise someone might remove them

Reply:
oh wait - its just the one below - but same comment